addons/default/visiosoft/profile-module/resources/views/address/list.twig

```diff
@@ -14,8 +14,8 @@
                         </a>
                     </th>
                     <th scope="col">{{ trans("visiosoft.module.profile::field.adress_name.name") }}</th>
-                    <th scope="col">{{ trans("visiosoft.module.profile::field.adress_first_name.name") }}</th>
-                    <th scope="col">{{ trans("visiosoft.module.profile::field.adress_last_name.name") }}</th>
+                    <th scope="col" class="d-none d-md-table-cell">{{ trans("visiosoft.module.profile::field.adress_first_name.name") }}</th>
```

Comment:
why do we hiding first name? @spektra2147  @profstyle1

Reply:
this is not just for mobile it's for desktop too? @spektra2147

Reply:
yes for mobile